lib/logMessages.ts

```diff
@@ -0,0 +1,5 @@
+const logMessages = {
+    incompatibleUpdateAlerts: (key: string, operation: string) => `A warning occurred while applying ${operation} for key: ${key}, Warning: Trying to set array to object or vice versa`,
```

Comment:
I think this error message could be improved to be more precise - "Trying to set an object to an array" or "Trying to merge and array into an object"